### DISCUSSION TITLE
ESP32C3 How to set off the onboard LED.

### DISCUSSION THREAD

**Discussion by romakrau** [arendst/Tasmota#22683] [Q&A]
Thankful for any hints.

Greeting Roman

**Comment by Noschvie** [comment]
Auto Configuration : 01Space_ESP32-C3-RGB

**Comment by sfromis** [comment]
Well, ESP32-C3 is just the chip. All the rest depends on details of what board the chip is used on, and what connections are there, including different types of leds connected to specific pins.
